init.rb

```diff
@@ -28,17 +28,17 @@
   description 'Plugin to generate and use issue templates for each project to assist issue creation.'
   version '0.1.1'
   author_url 'http://twitter.com/akiko_pusu'
-  requires_redmine :version_or_higher => '2.5'
+  requires_redmine version_or_higher: '2.5'
   url 'https://bitbucket.org/akiko_pusu/redmine_issue_templates'
 
-  menu :admin_menu, :redmine_issue_templates, { :controller => 'global_issue_templates', :action => 'index' },
-       :caption => :global_issue_templates
+  menu :admin_menu, :redmine_issue_templates, { controller: 'global_issue_templates', action: 'index' },
+       caption: :global_issue_templates
 
   project_module :issue_templates do
-    permission :edit_issue_templates, {:issue_templates => [:new, :edit, :destroy, :move]}
-    permission :show_issue_templates, {:issue_templates => [:index, :show, :load, :set_pulldown]}
+    permission :edit_issue_templates, { issue_templates: [:new, :edit, :destroy, :move] }
+    permission :show_issue_templates, { issue_templates: [:index, :show, :load, :set_pulldown] }
     permission :manage_issue_templates, 
-      {:issue_templates_settings => [:show, :edit]}, :require => :member
+      { issue_templates_settings: [:show, :edit] }, require: :member

```

Comment:
__[rubocop]__
* Align the parameters of a method call if they span more than one line. [:ref](https://github.com/bbatsov/ruby-style-guide#no-double-indent)